main.go

```
@@ -1,7 +1,52 @@
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
+	supa "github.com/supabase-community/supabase-go"
+)
+
+type Config struct {
+	port           string
```

golangci-lint (1.62.2)

field `port` is unused

(unused)